simulator/wdc-simulator.js

```diff
@@ -432,28 +429,28 @@
       var dataGatheringStateInProgress = isInProgress
                                       && wdcCommandSim.state.currentPhase === WdcCommandSimulator.Phase.GATHER_DATA;
 
+      var isWDCUrlEmpty = (this.state.wdcUrl == '') ? true : false;
```

Comment:
The ternary operator is redundant here because the `==` will result in either `true` or `false`.

You also almost always want to use `===` instead of `==` in JavaScript because because `==` has weird matching behavior.  Check out this chart at [http://dorey.github.io/JavaScript-Equality-Table/unified/](http://dorey.github.io/JavaScript-Equality-Table/unified/)